From b7be80db5217aac86e84d64bf94407f11b6b2635 Mon Sep 17 00:00:00 2001
From: Tomas Chvatal <tomas.chvatal@gmail.com>
Date: Wed, 24 Jul 2013 23:29:11 +0200
Subject: [PATCH] Include iostream to avoid build failures
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes following crash:
In file included from
/var/tmp/portage/app-admin/zypper-1.9.1/work/zypper-1.9.1/src/utils/messages.cc:10:0:
/var/tmp/portage/app-admin/zypper-1.9.1/work/zypper-1.9.1/src/Zypper.h:37:12:
error: ‘std::cout’ has not been declared
/var/tmp/portage/app-admin/zypper-1.9.1/work/zypper-1.9.1/src/Zypper.h:38:12:
error: ‘std::cerr’ has not been declared
---
 src/Zypper.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Zypper.h b/src/Zypper.h
index aef000f..feff271 100644
--- a/src/Zypper.h
+++ b/src/Zypper.h
@@ -34,6 +34,7 @@
 using namespace zypp;
 
 // Convenience
+#include <iostream>
 using std::cout;
 using std::cerr;
 using std::endl;
-- 
1.8.1.5

